evalml/automl/pipeline_search_plots.py

```diff
@@ -47,7 +47,7 @@ def update(self, results, objective):
         if len(results["search_order"]) > 0 and len(results["pipeline_results"]) > 0:
             iter_idx = results["search_order"]
             pipeline_res = results["pipeline_results"]
-            iter_scores = [pipeline_res[i]["mean_cv_score"] for i in iter_idx]
+            iter_scores = [pipeline_res[i]["validation_score"] for i in iter_idx]
```

Comment:
Moving towards plotting the validation score rather than the CV score for the search iteration plot.